pkg/cert/kube_certs.go

```diff
@@ -196,15 +197,20 @@ func NewMetaData(certPATH, certEtcdPATH string, apiServerIPAndDomains []string,
 	data.DNSDomain = DNSDomain
 	data.APIServer.IPs = make(map[string]net.IP)
 	data.APIServer.DNSNames = make(map[string]string)
-	_, svcSubnet, err := net.ParseCIDR(SvcCIDR)
-	if err != nil {
-		return nil, errors.Wrapf(err, "unable to parse ServiceSubnet %v", SvcCIDR)
-	}
-	svcFirstIP, err := utilnet.GetIndexedIP(svcSubnet, 1)
-	if err != nil {
-		return nil, err
+
+	for _, cidr := range strings.Split(SvcCIDR, ",") {
```

Comment:
Same issue, this should be validated at the very beginning part, or validation layer. It is not suggested that we validate this in the underlying implementation layer. Should use `Defensive Programming`.